src/main/java/io/github/apace100/apoli/mixin/LivingEntityMixin.java

```diff
@@ -480,6 +480,20 @@ private void modifyFlySpeed(CallbackInfoReturnable<Float> cir) {
         cir.setReturnValue(PowerHolderComponent.modify(this, ModifyAirSpeedPower.class, cir.getReturnValue()));
     }
 
+    @Unique
+    private Entity apoli$invisibilityActor;
```

Comment:
There is no need for this field and caching the argument to the `getAttackDistanceScalingFactor` call.
In a redirect, in addition to the object the method is called on and the method's arguments, you can also specify the arguments to the containing method afterwards. So you could just change line 492 to `private boolean invisibilityException(LivingEntity instance, Entity entity) {` and it would take it from there. :)